docs/basics/update.md

```diff
@@ -8,7 +8,7 @@ Similar to traditional databases, RisingWave supports data updates and deletions
 
 A few details worth noting:
 
-* Data on a table can be updated or deleted, even if the table was created using the `create table ... with ...`` statement. Of course, since RisingWave does not support transactions, users need to be extra careful with data accuracy;
+* Data on a table can be updated or deleted, even if the table was created using the `create table ... with ...` statement. Of course, since RisingWave does not support transactions, users need to be extra careful with data accuracy;

```

Comment:
```suggestion
* Data on a table can be updated or deleted, even if the table was created using the `CREATE TABLE ... WITH ...` statement. Of course, since RisingWave does not support transactions, users need to be extra careful with data accuracy;

```